src/transformers/processing_utils.py

```diff
@@ -128,6 +138,12 @@ class ImagesKwargs(TypedDict, total=False):
     class methods and docstrings.
 
     Attributes:
+        annotations (`AnnotationType` or `List[AnnotationType]`, *optional*):
+            List of annotations associated with the image or batch of images.
+        return_segmentation_masks (`bool`, *optional*):
+            Whether to return segmentation masks.
+        masks_path (`str` or `pathlib.Path`, *optional*):
+            Path to the directory containing the segmentation masks.
         do_resize (`bool`, *optional*):
```

Comment:
@zucchini-nlp I can make it to this format. However, I thought the intention of this uniform kwargs was to use single ImagesKwargs to use in every other processor since the variable is can be `Optional`.
Also it is quite difficult to differentiate that these values are specific to DINO-only.

@EduardoPach Any advice for to which variable to move in this `ModelImagesKwargs`?

Reply:
General `ImageKwargs` should hold kwargs that are used in almost all models, while `annotations` are used only in some object detection models. Yes, the way we have it now looks good to me

Regarding the tests, we can't merge the PR if tests are failing. So feel free to make changes to the test files to make the CI happy. After that we can request review from core maintainer

Reply:
@zucchini-nlp Yeah this is still on draft! (I will ping again the PR request from github)